packages/venia-ui/lib/components/MyAccount/myAccount.css

```diff
@@ -1,15 +1,6 @@
 .root {
     display: grid;
-    gap: 1rem;
-    padding: 1.5rem;
-}
-
-.user {
-    display: grid;
-    gap: 0.5rem;
     grid-auto-flow: row;
-}
-
-.subtitle {
-    color: rgb(var(--venia-global-color-text-alt));
+    padding: 1.5rem 1.5rem;
```

Comment:
```suggestion
    padding: 1.5rem;
```

Reply:
Resolved in https://github.com/magento/pwa-studio/pull/2550/commits/e72d38e5af785a2ce2731f7a31cc2709ff23c2d1.